services/course-material/src/components/chatbot/Chatbot/ChatbotReferenceList.tsx

```diff
@@ -1,8 +1,9 @@
 import { css } from "@emotion/css"
```

Comment:
move the popover list rendering into its own component (and that component will render the components for the individual popovers.